src/UI/Factory.php

```diff
@@ -336,4 +336,63 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      Drop Downs reveal a list of interactions that change the system’s status or navigate to
+	 *      a different view.
+	 *   composition: >
+	 *      Drop Down is a clickable, graphically obtrusive control element. It can
+	 *      bear text.
+	 *   effect: >
+	 *      On-click, a list of actions is revealed. Clicking an item will trigger the action indicated.
+	 *      Clicking outside of an opened drop down will close the list of items.
```

Comment:
I did not test it, but my understanding was that the Triggerer traits in DropdownItem will allow this. Not sure if the implementation is already correct here.